test/example.c

```diff
@@ -35,6 +35,11 @@ static const char hello[] = "hello, hello!";
 static const char dictionary[] = "hello";
 static unsigned long dictId = 0; /* Adler32 value of the dictionary */
 
+
+void test_compress      (unsigned char *compr, z_size_t comprLen,
```

Comment:
Unneccessary split lines. We use 120 char long lines in zlib-ng

Reply:
I just moved that code from down below without modifying the whitespace.

Reply:
Now I have added another commit that will fix the split lines.